src/FileConventions.Test/FileConventions.Test.fs

```diff
@@ -78,3 +78,21 @@ let DetectAsteriskInPackageReferenceItems1 () =
 let DetectAsteriskInPackageReferenceItems2 () =
     let fileInfo = (FileInfo (Path.Combine(__SOURCE_DIRECTORY__, "DummyFiles", "DummyFsprojWithoutAsterisk.fsproj")))
     Assert.That(DetectAsteriskInPackageReferenceItems fileInfo, Is.EqualTo false)
+
+
+[<Test>]
+let MissingVersionsInNugetPackageReferencesTest1 () =
+    let fileInfo = (FileInfo (Path.Combine(__SOURCE_DIRECTORY__, "DummyFiles", "DummyWithMissingVersionsInNugetPackageReferences.fsx")))
+    Assert.That(DetectMissingVersionsInNugetPackageReferences fileInfo, Is.EqualTo true)
+
+
+[<Test>]
+let MissingVersionsInNugetPackageReferencesTest2 () =
+    let fileInfo = (FileInfo (Path.Combine(__SOURCE_DIRECTORY__, "DummyFiles", "DummyWithoutMissingVersionsInNugetPackageReferences.fsx")))
+    Assert.That(DetectMissingVersionsInNugetPackageReferences fileInfo, Is.EqualTo false)
+
+
+[<Test>]
+let MissingVersionsInNugetPackageReferencesTest3 () =
+    let fileInfo = (FileInfo (Path.Combine(__SOURCE_DIRECTORY__, "DummyFiles", "DummyWithoutNugetPackageReferences.fsx")))

```

Comment:
@realmarv why are you not using dummyFilesDirectory in your tests?

Reply:
Done